### PR TITLE
{ServiceBus} `az servicebus queue update`: Add breaking change alert messages

### PR DESCRIPTION
**Pull request by damodaravadhani**
**Related command**
az servicebus queue update

**Description**
Adding breaking change alert messages wherever required

**Testing Guide**

**History Notes**

{ServiceBus}  `az servicebus queue update`: Add breaking change alerts

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [X] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
ServiceBus

**Comment by damodaravadhani** [comment]
@evelyn-ys, we are removing certain properties from `queue update` cmdlets in a future release since the properties cannot be updated once created. Also, since we are planning to migrate to entire repo to aaz, timespan inputs too would not accept values in the format dd:mm:ss. Added a warning for this too.

**Review by evelyn-ys — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/servicebus/custom.py [hunk 1]
```diff
@@ -635,6 +635,7 @@ def return_valid_duration(update_value, current_value=None):
     from isodate import Duration
     from azure.cli.core.azclierror import InvalidArgumentValueError
     from azure.cli.command_modules.servicebus.constants import DURATION_SECS, DURATION_MIN, DURATION_DAYS
+    from argcomplete import warn
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/servicebus/custom.py:638]: Why using `warm` from `argcomplete`. This usage is weird.

File: src/azure-cli/azure/cli/command_modules/servicebus/custom.py [hunk 2]
```diff
@@ -673,6 +675,7 @@ def return_valid_duration_create(update_value):
     from datetime import timedelta
     from isodate import parse_duration
     from knack.util import CLIError
+    import warnings
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/servicebus/custom.py:678]: It is not Azure CLI convention to use `warnings` module. Instead, warnings in Azure CLI is written like

https://github.com/Azure/azure-cli/blob/d331edf76e5866cda1e6e9641d935256fc4f03ff/src/azure-cli/azure/cli/command_modules/profile/custom.py#L38-L39

**Pull request merged**